zgw/catalogi-api/src/main/kotlin/com/ritense/catalogiapi/CatalogiApiPlugin.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.catalogiapi
+
+import com.ritense.catalogiapi.client.ZaaktypeInformatieobjecttypeRequest
+import com.ritense.catalogiapi.domain.Informatieobjecttype
+import com.ritense.catalogiapi.domain.ZaaktypeInformatieobjecttype
+import com.ritense.plugin.annotation.Plugin
+import com.ritense.plugin.annotation.PluginProperty
+import com.ritense.zakenapi.client.CatalogiApiClient
+import com.ritense.zgw.Page
+import mu.KotlinLogging
+import java.net.URI
+
+@Plugin(
```

Comment:
Cool new plugin. Will the FE be alright when it retrieves this new unknown plugin?

Reply:
If it's not supported it will not be shown. But Maxim is creating the FE for this.